platform/README.md

```diff
@@ -1,28 +1,29 @@
 # New platform
 
 ## Dev setup
-
-Make sure you're running the Node version specified in `../.node-version`.
+While in this directory, make sure you're running the Node version specified in `../.node-version`.
 
 If you've upgraded your npm version, you might need to delete your `node_modules`
 folder and re-run `npm install`.
 
 ## Running code
 
-Make sure to build the code first, e.g. `npm run ts:build` or `npm run ts:start`.
+While in this directory, make sure to build the code first, e.g. `npm run ts:build` or `npm run ts:start`.
```

Comment:
Hopefully this goes away very soon 🙈 But either way: you don't have to be in this directory, as the `ts:build` and `ts:start` commands are defined in the top-level `package.json`. You can run all the commands in this file from root instead of from `./platform/`.